service/index.js

```diff
@@ -171,8 +160,6 @@ app.get(/^\/v1\/polyfill(\.\w+)(\.\w+)?/, function(req, res) {
 	res.set('Content-Type', contentTypes[fileExtension]+';charset=utf-8');
 	res.set('Access-Control-Allow-Origin', '*');
 	res.send(op);
-	metrics.addResponseTime(Date.now() - responseStartTime);
```

Comment:
I think this should be end to end.  Currently there's nothing in between the start and stop points that has any chance of being slow - there's no IO of any kind, other than the final HTTP response.  The current data shows an average time of 2ms, while the total RTT will be more like 20 at a minimum.